shared/fs/banner/conflict-banner.tsx

```diff
@@ -0,0 +1,92 @@
+import * as React from 'react'
+import * as Kb from '../../common-adapters'
+import * as Flow from '../../util/flow'
+import * as Types from '../../constants/types/fs'
+
+export type Props = {
+  conflictState: Types.ConflictState
+  onStartResolving: () => void
+  onSeeOtherView: () => void
+  isUnmergedView: boolean
+  onFinishResolving: () => void
+  onFeedback: () => void
+  onHelp: () => void
+  tlfName: string
+}
+
+const getMessage = (tlf: string, conflictState: Types.ConflictState, isUnmerged): string => {
+  switch (conflictState) {
+    case Types.ConflictState.InConflictStuck:
+      return (
+        `Your changes to ${tlf} conflict with changes made to this` +
+        ' folder on another device. Automatic conflict resolution has failed,' +
+        ' so you need to manually resolve the conflict. This is not' +
+        ' supposed to happen!'
+      )
+    case Types.ConflictState.InConflictNotStuck:
```

Comment:
Yeah let's not show anything in this case.  This state _is_ supposed to happen, whenever there's a conflict.

Reply:
Cool, sorry perpetuating my misunderstanding @songgao. Will fix then resolve conflicts.